game/config.hh.in

```diff
@@ -9,3 +9,7 @@
 #define LIBXMLPP_VERSION_2_6 @LIBXMLPP_VERSION_2_6@
 #define LIBXMLPP_VERSION_3_0 @LIBXMLPP_VERSION_3_0@
 #define USE_BOOST_FS @USE_BOOST_FS@
+#define SIGCPP_VERSION_2_0 @SigC++_VERSION_2_0@
+#define SIGCPP_VERSION_3_0 @SigC++_VERSION_3_0@
+#define GLIBMM_VERSION_2_4 @Glibmm_VERSION_2_4@
+#define GLIBMM_VERSION_2_68 @Glibmm_VERSION_2_68@
```

Comment:
These defines aren't actually used within the code so i wasn't sure if i had to include them here or not